ecosystem/sep-0024.md

```diff
@@ -53,7 +53,7 @@ Authorization: Bearer <JWT>
 
 In the case of the interactive webapp, since the client cannot add the authorization header, we recommend passing a short-lived JWT via URL query parameters and then using your own backend session scheme for the rest of the interactive flow.  Query parameters can leak, so its important to have this JWT be one-time use, or at least short-lived.
 
-### Shared, Omnibus, or Pooled Accounts
+### Shared, Omnibus, or Muxed Accounts
```

Comment:
Using "Pooled" here is intentional, what is the reasoning for using muxed?

Reply:
Rolled back.